packages/niru/niru.1.0.0/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "1.0.0"
+synopsis: "A simiarity file comparator"
+description:
+  "niru is a simiarity file comparator. It computes the similary between two files by spliting them into k-blocks and uses the Jaccard index."
+maintainer: ["EruEri"]
```

Comment:
```suggestion
maintainer: ["EruEri"]
```
Can you add an email or a way to be contacted?

Reply:
Yes